src/services/jwt.service.js

```
@@ -0,0 +1,42 @@
+const jwt = require('jsonwebtoken');
+
+require('dotenv').config();
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, { expiresIn: '15m' });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
```

`verify` and `verifyRefresh` swallow verification errors and return `null` which is a valid and acceptable behavior for the app's error flow. As a small improvement, you could check token truthiness before calling `jwt.verify` to avoid unnecessary try/catch when `token` is falsy (although current code handles that by returning null). This is optional.